### PR TITLE
Candidate Release 2.1.0

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bamtofastq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/bamtofastq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
need #66

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,29 @@
...
+
+### `Fixed`
+
+- [#62](https://github.com/nf-core/bamtofastq/pull/62) Adjust subway map for dark mode.
+
+### `Dependencies`
+
+| Dependency | Old version | New version |
+| ---------- | ----------- | ----------- |
+|            |             |             |
```
Comment (maxulysse) [inline CHANGELOG.md:25]:
```suggestion
| MultiQC    | 1.14        | 1.15        |
```

**Review by FriederikeHanssen — APPROVED** [review]
Looks good, thanks a lot for taking care of the template updates 🙏

**Review by CarsonJM — APPROVED** [review]
Looks good to me as well!

**Pull request merged**